config/bot.yml

```diff
@@ -1,2 +0,0 @@
-SLACK_API_TOKEN: 'change me'
-bot_name: 'bot'
```

Comment:
Weird the comment should have been from 21 hours ago and the "approved" should have been from 11 minutes ago but it is showing as opposite for me.

I addressed the concern in: https://github.com/ImperialLabs/slapi/pull/31/commits/deae9a8017aad30d9365a280b16c39010ddf6397

I am now curious where my other comment went?